### ISSUE TITLE
Error when choosing custom persistence profile

### ISSUE THREAD

**Issue by grf5** [F5Networks/f5-appsvcs-templates#64]
### Environment
 * FAST RPM Version: 1.11.0
 * BIG-IP Version: 14.1.2.6

### Summary
Error when choosing a custom persistence profile in FAST ... validation requires strict values.

```
undefinied /test/fast_testabc_app/fast_test_abc_vs/persistenceMethods/0: should be equal to one of the allowed values [“cookie”,”destination-address”,”msrdp”,”source-address”,”tls-session-id”]
````

### Steps To Reproduce
1. Create custom persistence profile
2. Deploy FAST HTTPS Offload template with custom persistence profile referenced and view validation failure message.

![Custom persistence profile selected](https://user-images.githubusercontent.com/27083437/129076961-34b4c391-d0e8-466d-891c-35bf7464a5be.png)

![Validation error](https://user-images.githubusercontent.com/27083437/129076976-3227d968-0ab8-4eab-8784-038aabd617ca.png)

### Expected Behavior
FAST should accept the custom persistence profile selected.

### Actual Behavior
FAST validation requires a certain value for the persistence type: 

Original RFE: #48

**Comment by shyawnkarim** [comment]
Thanks for reporting this issue. We are now tracking it internally with ID MYSTIQUE-454.

**Comment by shyawnkarim** [comment]
Closing.

This bug fix was included with [Release 1.12.0](https://github.com/F5Networks/f5-appsvcs-templates/releases/tag/v1.12.0).
